tutorials/en_us/JARS/ADVANCED.md

```diff
@@ -2,45 +2,67 @@
 
 If you want to go a little more in-depth with JAR picking, you've come to the right place.
 
-## Most popular Spigot/Bukkit JARs (PLUGINS):
+## Server Software:
 
-### Recommended top picks:
+Server software is the software that allows multiple players to play on a server together.
+
+### Plugins:
+
+- [CraftBukkit](https://getbukkit.org/download/craftbukkit) - It's the BASE for all forks that are out today and they still are updating it to this day, not newbie friendly at all because it requires manual building (or you can get it [here](https://getbukkit.org)). The only real case of use is for Minecraft versions from 1.0.0 to 1.4.5 as there is no spigot release for those versions.
```

Comment:
```suggestion
- [CraftBukkit](https://getbukkit.org/download/craftbukkit) - It's the BASE for all forks that are out today and they still are updating it to this day, not newbie friendly at all because it requires manual building (or you can get it [here](https://getbukkit.org)). The only real use is for Minecraft versions from `1.0.0` to `1.4.5` as there are no Spigot release for these versions.
```